variant-caller-wdl-checker/topmed_freeze8_caller_checker.wdl

```diff
@@ -0,0 +1,257 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/DataBiosphere/topmed-workflow-variant-calling/1.0.0/variant-caller-wdl/topmed_freeze8_caller.wdl" as TopMed_variantcaller
+
+
+workflow checkerWorkflow {
+  input {
+    File inputTruthVCFFile
+
+    String? docker_image
+    String docker_concordance_image = "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.3.2-1510681135"
+
+    Array[File]? input_crai_files
+    Array[File] input_cram_files
+
+    File referenceFilesBlob
+  }
+
+   call TopMed_variantcaller.TopMedVariantCaller as variantcaller {
+     input:
+       input_crai_files = input_crai_files,
+       input_cram_files = input_cram_files,
+       docker_image = docker_image,
+       referenceFilesBlob = referenceFilesBlob
+   }
+
+  call checkerTask {
+      input:
+          inputTruthVCFFile = inputTruthVCFFile,
+          inputTestVCFFile = variantcaller.topmed_variant_caller_output,
+          docker_concordance_image = docker_concordance_image,
+          genomeReferenceFileNameAndPath = "resources/ref/hs38DH.fa",
+          referenceFileBlob = referenceFilesBlob,
+          reference_disk_size = 20.0
+  }
+
+  meta {
+      author : "Walt Shands"
+      email : "jshands@ucsc.edu"
+      description: "This is the checker workflow WDL for U of Michigan's [TOPMed Freeze 8 Variant Calling Pipeline](https://github.com/statgen/topmed_variant_calling)"
+   }
+
+}
+
+
+task checkerTask {
+  input {
+      File inputTruthVCFFile
+      File inputTestVCFFile
+
+      File referenceFileBlob
+      Float reference_disk_size
+
+      String genomeReferenceFileNameAndPath
+      String docker_concordance_image
+
+      Int additional_disk = 20
+  }
+
+  Float disk_size = reference_disk_size + size(inputTruthVCFFile, "GB") + size(inputTestVCFFile, "GB") + additional_disk
+
+  command <<<
+    python3 <<CODE
```

Comment:
I don't really understand this, but why is this python3, whereas above is python3.5? And there is also a Python 2.7 image being used? This may be constrained by the docker images you are using, but off-hand it feels like it would be nice if the same Python version were being used.

Reply:
changed python 2.7 to python 3.5 in the variant caller. However for the checker the python version does depend on which versions of python are installed in the image and since different images are used for the variant caller and the concordance code in the checker the python versions are different.